com.unity.netcode.gameobjects/Runtime/Components/NetworkAnimator.cs

```diff
@@ -489,46 +649,36 @@ public void NetworkSerialize<T>(BufferSerializer<T> serializer) where T : IReade
             }
         }
 
-        [SerializeField] private Animator m_Animator;
-
-        public Animator Animator
-        {
-            get { return m_Animator; }
-            set
-            {
-                m_Animator = value;
-            }
-        }
-
-        internal bool IsServerAuthoritative()
+        /// <summary>
+        /// Determines whether the <see cref="NetworkAnimator"/> is <see cref="AuthorityModes.Server"/> or <see cref="AuthorityModes.Owner"/> based on the <see cref="AuthorityMode"/> field.
+        /// Optionally, you can still derive from <see cref="NetworkAnimator"/> and override the <see cref="OnIsServerAuthoritative"/> method.
+        /// </summary>
+        /// <returns><see cref="true"/> or <see cref="false"/></returns>
+        public bool IsServerAuthoritative()
```

Comment:
Are we sure we want to make this public?

Reply:
Yeah, I think this follows the same pattern as NetworkTransform.